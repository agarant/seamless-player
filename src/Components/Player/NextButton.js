import React from "react";
import styled from "styled-components";
import NextIcon from "material-ui-icons/SkipNext";
import IconButton from "material-ui/IconButton";

const Button = styled(IconButton)`
  color: #616161 !important;
  margin-left: 24px;
  font-size: 18px;
`;

export default ({ onClick, disabled }) => (
  <Button onClick={onClick} disabled={disabled}>
    <NextIcon />
  </Button>
);
