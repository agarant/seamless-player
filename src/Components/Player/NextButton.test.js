import React from "react";
import renderer from "react-test-renderer";
import NextButton from "./NextButton";

test("<NextButton /> renders the proper icon when playing", () => {
  const tree = renderer.create(<NextButton disabled={false} />).toJSON();
  expect(tree).toMatchSnapshot();
});
