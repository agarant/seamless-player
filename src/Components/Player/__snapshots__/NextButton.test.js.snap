// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NextButton /> renders the proper icon when playing 1`] = `
<button
  className="MuiButtonBase-root-10 MuiIconButton-root-1 sc-bdVaJa ikOtYg"
  disabled={false}
  onBlur={[Function]}
  onClick={undefined}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  role="button"
  tabIndex={0}
  type="button"
>
  <span
    className="MuiIconButton-label-7"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root-12 MuiIconButton-icon-8"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M6 18l8.5-6L6 6v12zM16 6v12h2V6h-2z"
      />
    </svg>
  </span>
  <span
    className="MuiTouchRipple-root-13"
  />
</button>
`;
